dom.go

```diff
@@ -404,6 +404,8 @@ func wrapHTMLElement(o *js.Object) HTMLElement {
 		return &HTMLUnknownElement{BasicHTMLElement: el}
 	case js.Global.Get("HTMLVideoElement"):
 		return &HTMLVideoElement{HTMLMediaElement: &HTMLMediaElement{BasicHTMLElement: el}}
+	case js.Global.Get("HTMLTemplateElement"):
```

Comment:
Let's move this entry so it's more alphabetical in this list of cases. It should be between HTMLTableSectionElement and HTMLTextAreaElement.